code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -831,14 +845,13 @@
 			return
 		switching = 0
 		var/N = rand(1,3)
-		if(get_turf(user))
-			switch(N)
-				if(1)
-					playsound(get_turf(user), 'sound/weapons/genhit1.ogg', 50, 1)
-				if(2)
-					playsound(get_turf(user), 'sound/weapons/genhit2.ogg', 50, 1)
-				if(3)
-					playsound(get_turf(user), 'sound/weapons/genhit3.ogg', 50, 1)
+		switch(N)
```

Comment:
God only knows why these lost their get turf check. I'll give those back in the next commit.